app/src/pages/organization/organizationProjectsPage/projectsListTable/projectsListTable.scss

```diff
@@ -69,3 +70,8 @@
 .loader {
   margin: auto;
 }
+
+.projects-pagination-wrapper {
+  min-height: 100%;
+  flex: 1;
+}
```

Comment:
Does it make sense to hold this styles in the `PaginationWrapper`?
Looks like they are repeated on every page.

Reply:
I thought, in this case, it seems much more customizable, and the PaginationWrapper could be used not only for full height but also in any section